src/browser/browser.ts

```diff
@@ -351,8 +351,8 @@ export class Browser {
               return totalPanelsRendered >= panelCount;
             }
 
-            const panelCount = document.querySelectorAll('.panel').length || document.querySelectorAll('.panel-container').length;
-            return (window as any).panelsRendered >= panelCount || (window as any).panelsRendered === undefined;
+            const panelCount = document.querySelectorAll('.panel-solo').length || document.querySelectorAll('[class$=\'panel-container\']').length;
```

Comment:
why did it change? Does it work right now?

Reply:
I think it changes a while ago in Grafana but we didn't notice because it was 0 and `panelsRendered` is `undefined` until the panel is rendered. So for one panel (what we always render because the condition for full page is different), we checked `undefined >= 0` for a while (which is false) and when the panel is rendered, we checked `0 >= 0` (which is true) so it returned. 

I don't think we actually need this and we could set it to 1 but I would need to think more about it before making this more bigger change (and it would be nice to have a test suite as well for this kind of things).